### ISSUE TITLE
Travis CI builds fail due to Rails 5.1 release

### ISSUE THREAD

**Issue by mattbrictson** [bootstrap-ruby/rails-bootstrap-forms#329]
Rails 5.1 final has been released, and our Travis CI build is automatically picking it up.

This causes problems because certain deprecated features have been completely removed in Rails 5.1. Among those are non-versioned migrations. Rails 5.1 now requires that every migration specify an explicit Rails version number.

```
Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:
  class CreateUsers < ActiveRecord::Migration[4.2]
```

Failing build:

https://travis-ci.org/bootstrap-ruby/rails-bootstrap-forms/builds/227443604

**Comment by mattbrictson** [comment]
@koenpunt would you be able help on this build issue? I think you probably have the most knowledge of how our Appraisal tests are set up (see #278).

**Comment by koenpunt** [comment]
Sure, I can do that later today

**Comment by mattbrictson** [comment]
I'll try to fix this later this week. If anyone else wants to take a crack at it in the meantime, let me know! This is fairly high priority since it is breaking our Travis builds.

**Comment by koenpunt** [comment]
@mattbrictson totally forgot about this. Made a start in #330

**Comment by mattbrictson** [comment]
Fixed via #330
